lib/types/options.d.ts

```diff
@@ -60,6 +60,10 @@ export type Options = {
   gitHubPassword: string | undefined;
 };
 
+export type OptionsForRunningElmReview = Omit<Options, 'elmJsonPath'> & {
+  elmJsonPath: Path;
+};
+
```

Comment:
If we think more types are going to have to change for running elm-review, we could also abstract this a little:

```ts
type OverriddenFieldsForRunningElmReview = { elmJsonPath: Path, ... }

export type OptionsForRunningElmReview = Omit<Options, keyof OverriddenFieldsForRunningElmReview> & OverriddenFieldsForRunningElmReview
```

Also, this name is pretty long, but I couldn't think of anything better 😅